acados/ocp_nlp/ocp_nlp_constraints_bgh.c

```diff
@@ -158,6 +313,80 @@ void *ocp_nlp_constraints_bgh_model_assign(void *config, void *dims_, void *raw_
     return model;
 }
 
+int ocp_nlp_constraints_bgh_bounds_set(void *config_, void *dims_,
```

Comment:
`ocp_nlp_constraints_bgh_model_set` is a more generic naming that would work also for e.g. matrices

Reply:
by the way, are there setters for the matrices C, D and nonlinear function h too?